patches/server/0853-Kick-on-main-for-illegal-chars.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Nassim Jahnke <jahnke.nassim@gmail.com>
+Date: Mon, 17 Jan 2022 19:47:19 +0100
+Subject: [PATCH] Kick on main for illegal chars
+
+
+diff --git a/src/main/java/net/minecraft/server/network/ServerGamePacketListenerImpl.java b/src/main/java/net/minecraft/server/network/ServerGamePacketListenerImpl.java
+index 926d0a80cbb55184955ac6720948d2e86683cc57..659777ccc9beb1833849971f7423b1c33ad55473 100644
+--- a/src/main/java/net/minecraft/server/network/ServerGamePacketListenerImpl.java
++++ b/src/main/java/net/minecraft/server/network/ServerGamePacketListenerImpl.java
+@@ -2067,7 +2067,11 @@ public class ServerGamePacketListenerImpl implements ServerPlayerConnection, Ser
+ 
+         for (int i = 0; i < s.length(); ++i) {
+             if (!SharedConstants.isAllowedChatCharacter(s.charAt(i))) {
++                // Paper start - push to main for event firing
++                server.scheduleOnMain(() -> {
```

Comment:
```suggestion
+                this.server.scheduleOnMain(() -> {
```

Also should just do 2 paper commands on the lines changed instead of a start/end